versioned_docs/version-2.x/magic-button/storefront-view.mdx

```diff
@@ -0,0 +1,51 @@
+---
+sidebar_position: 4
+title: StoreFront View
+description:
+  Enhances your stores functionality by providing an edit button alongside each
+  DOM element, with each content source clearly displayed, content creators can
+  directly access the source for editing, thereby improving content management
+  efficiency.
+---
+
+import BrowserWindow from "@site/src/components/BrowserWindow";
+import SinceVersion from "@site/src/components/SinceVersion";
+import ContactLink from "@site/src/components/ContactLink";
+
+<SinceVersion tag="2.25" />
+
+The **Front-Store View** feature enhances your stores functionality by providing
+an edit button alongside each DOM element, with each content source clearly
+displayed, content creators can directly access the source for editing, thereby
+improving content management efficiency.
+
+:::caution WIP
+
+The content below is currently being written. If you need more detailed
+information, please <ContactLink />. We will make sure to answer you in a timely
+manner.
+
+:::
```

Comment:
**question:** now that the feature is done, should we do it in this PR or in an upcoming one?

Reply:
done in this PR (for what is ready)